lib/screens/login/login_screen.dart

```diff
@@ -110,6 +114,31 @@ class _LoginScreenState extends State<LoginScreen>
 
   @override
   Widget build(BuildContext context) {
+    ref.listen<AsyncValue<void>>(loginViewModelProvider, (_, next) {
```

Comment:
(Optional) It would be nice if we can move this block to another function, for example: `_setupListener()`